frontend/src/Editor/DataSourceManager/DataSourceManager.jsx

```diff
@@ -550,7 +550,16 @@ class DataSourceManagerComponent extends React.Component {
           animation={false}
           onExit={this.onExit}
         >
-          <Modal.Header>
+          <Modal.Header className="justify-content-start">
+            {selectedDataSource && (
+              <div
+                className={`back-btn ${this.props.darkMode ? 'dark' : ''}`}
+                style={{ marginRight: '1rem', cursor: 'pointer' }}
```

Comment:
```suggestion
                className={`back-btn me-3 ${this.props.darkMode ? 'dark' : ''}`}
                role="button"
```

Instead of inline-styles, we can use the existing classes